[ERROR-SonarQube version: 8.5.0] Value of parameter 'facets' (fileUuids) must be one of..

#### Describe the bug

Not able to run this on SonarQube 8.5.0, here's the output:

SonarQube online: true
Detected SonarQube version: 8.5.0
[ERROR] Value of parameter 'facets' (fileUuids) must be one of: [projects, moduleUuids, files, assigned_to_me, severities, statuses, resolutions, rules, assignees, authors, author, directories, scopes, languages, tags, types, owaspTop10, sansTop25, cwe, createdAt, sonarsourceSecurity]

#### User environment 

* OS: Centos 8

--- a/src/main/resources/requests.properties
+++ b/src/main/resources/requests.properties
@@ -44,7 +44,7 @@ GET_QUALITY_PROFILES_PROJECTS_REQUEST = %s/api/qualityprofiles/projects?key=%s
 # Request to get the list of projects linked to a profile in SQ 5.X
 GET_PROJECT_QUALITY_PROFILES_REQUEST = %s/api/qualityprofiles/search?projectKey=%s
 # Request to get the list of issues linked to a project
-GET_ISSUES_REQUEST = %s/api/issues/search?projects=%s&facets=types,rules,severities,directories,fileUuids,tags&ps=%d&p=%d&additionalFields=rules,comments&resolved=%s&branch=%s
+GET_ISSUES_REQUEST = %s/api/issues/search?projects=%s&facets=types,rules,severities,directories,files,tags&ps=%d&p=%d&additionalFields=rules,comments&resolved=%s&branch=%s
 # Request to get the list of a project's facets
 GET_FACETS_REQUEST = %s/api/issues/search?projects=%s&resolved=false&facets=rules,severities,types&ps=1&p=1&branch=%s
 # Request to get the list of a project's facets

